### ISSUE TITLE
Ability to add events to top level Modal container or onScroll event

### ISSUE THREAD

**Issue by Lyubomyr** [react-bootstrap/react-bootstrap#5638]
## Is your feature request related to a problem? Please describe
I need to know when the user is scrolling Modal content. This will help to add additional styles for scrolled content and do other changes. And currently, I can't see any possibility to do that.

## Describe the solution you'd like

It would be great to have some kind of popperConfig for Modal, where an object with events (e.g. {onScroll: methodName}) could be passed.

## Describe alternatives you've considered

Or at least have an onScroll event available for Modal.

## Additional context
There is no possibility to add event to top level container of Modal:
<img width="1084" alt="Screenshot 2021-02-01 at 19 04 49" src="https://user-images.githubusercontent.com/783229/106492582-c108a600-64c0-11eb-9436-9b622cfd965c.png">

**Comment by kyletsang** [comment]
Some props are forwarded to the underlying `.modal-dialog` element.  Could you use an `onScroll` there?

**Comment by Lyubomyr** [comment]
Yes, it would be great if `onScroll` also would be forwarded from the Modal element. But currently, it's not.
